src/components/@shared/FormFields/Nft/TxFee.tsx

```diff
@@ -55,7 +55,7 @@ export default function TxFee({
   return gasFee ? (
     <p>
       Gas fee estimation for this artwork
-      <Conversion price={gasFee} />
+      <Conversion price={gasFee} symbol={'OCEAN'} />
```

Comment:
this doesn't seem ok, what's the relation between gas fee and ocean?

Reply:
this is how we do currency calculations in codebase, always converting from OCEAN value, which the `gasFee` is in. The `Conversion` component needs to know the value (`price`) and the denomination of the value (`symbol`).

Agree this isn't super clear and was thinking to use different named props like `price` & `priceSymbol` to make connection more clear.

But in this specific case, for the gas fee, we should denominate in ETH, so could refactor this file a bit where we convert fetched gas fee to OCEAN somewhere above

Reply:
oh, didn't imagine  it was this complicated. Maybe leave it for another pr since it's not really related to this one.

Reply:
alse realized I closed the other gas fee estimation PRs so even tending towards removing estimation from here too, as the same argument applies here: nobody uses ETH mainnet and estimations are only useful for this network

Reply:
just removed it for now as same applies as for other gas fee estimation implementations: we can go back to it once we have ETH mainnet usage